client/components/ReviewRating/ReviewList/ReviewBody.jsx

```diff
@@ -20,14 +20,18 @@ const ReviewBody = ({ review }) => {
       {isMoreThanMaxChar ? (
         <div>
           {review.body.slice(0, 250)}
-          <button type="button" onClick={() => setIsMoreThanMaxChar(false)}>show more</button>
+          ...
```

Comment:
I'm not quite sure what all this markup is for but it seems strange. Can you explain what you're trying to do here?